docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -9,9 +9,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 73,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The autoreload extension is already loaded. To reload it, use:\n",
+      "  %reload_ext autoreload\n"
```

Comment:
De autoreload zou ik gebruiken voor als je zelf codeert in notebooks, maar niet voor notebooks die gebruikers gaan draaien of die gebruikt worden in docs. Dan is het reloaden overbodig omdat het draait op de release branch en niet op lokale veranderingen.